src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -79,6 +79,17 @@ export interface SavedObjectsBulkResponse<T = unknown> {
   saved_objects: Array<SavedObject<T>>;
 }
 
+/**
+ *
+ * @public
+ */
+export interface SavedObjectsFindResult<T = unknown> extends SavedObject<T> {
+  /**
+   * The ES search's `_score` of this result.
+   */
+  score: number;
```

Comment:
Shouldn't we keep the underscore convention `_score` otherwise we could get a conflict if there was ever a "score" saved object type which a plugin author won't realize until they try to do a find. I also think the underscore makes it more obvious that this is ES "metadata" instead of a root property. (As an aside, we should probably have a saved object type blacklist)

Reply:
> otherwise we could get a conflict if there was ever a "score" saved object type 

The added `score` property is at the root level of the `SavedObject`, not in the `attributes`, so I think we are fine here?

Reply:
Ah yes, we serialize the type property into `attributes` :+1: